sql/moz-fx-data-shared-prod/search_terms/suggest_impression_sanitized/view.sql

```diff
@@ -4,4 +4,4 @@ AS
 SELECT
   * REPLACE (mozfun.norm.metadata(metadata) AS metadata)
 FROM
-  `moz-fx-data-shared-prod.search_terms_derived.suggest_impression_sanitized_v1`
+  `moz-fx-data-shared-prod.search_terms_derived.suggest_impression_sanitized_v2`
```

Comment:
```suggestion
  `moz-fx-data-shared-prod.search_terms_derived.suggest_impression_sanitized_v1`
```

I'm going to defer this to a new PR in January, both because it changes the view interface and because it will require reauth.